docs/running-workflows/launching-workflows/index.md

```diff
@@ -135,22 +135,13 @@ The standard set of badges look as follows:
 
 ![Launch with Snakemake REANA at CERN](https://www.reana.io/static/img/badges/launch-with-snakemake-on-reana-at-cern.svg)
 
-If you would like to generate your own custom badge, you can use external services such as [shields.io](https://shields.io) or [badgen.net](https://badgen.net).
```

Comment:
I think we should keep the part about generating custom badge designs using external services, as the badge creator is not a substitute for it (e.g. there is no way to change the text inside the badge, etc.)